### ISSUE TITLE
Transparent Token

### ISSUE THREAD

**Issue by stphnwlkr** [uswds/uswds#3934]
There doesn't appear to be a token to set the color as transparent. This would be extrememly helpful in the theme setting related to background colors for breadcrumbs, cards, etc. This should work but doesn't: 

$theme-breadcrumb-background-color: "transparent"

**Comment by mejiaj** [comment]
The background is set on breadcrumb so we can set an accessible color on the breadcrumb background.

```scss
// _breadcrumb.scss
.usa-breadcrumb {
   ⋯
  @include set-text-and-bg($theme-breadcrumb-background-color);
}
```

A `transparent` shortcode exists in [`_system-tokens.scss`](https://github.com/uswds/uswds/blob/develop/src/stylesheets/core/_system-tokens.scss#L1153)

```scss
$system-color-shortcodes: (
  "transparent": transparent,
   ⋯
```

The function that is causing the error is `get-color-from-bg()`. This brings up the question as to how we would handle comparing link colors against a transparent background.

Solution could be a warning similar to browsers saying we can't determine the contrast, but allow the fallback color to be set.

CC @thisisdano

**Comment by stphnwlkr** [comment]
Since a transparent background is the same as no background, it should be tested against the default body background color.

**Comment by thisisdano** [comment]
So we made some changes that will start to address your issue in https://github.com/uswds/uswds/pull/3949:

First, we added a few new settings:

Setting | Description | Default
--- | --- | ---
$theme-body-background-color | Sets the background of the `body` element | "white"
$theme-text-color | Site text color for light backgrounds | "ink"
$theme-text-reverse-color | Site text color for dark backgrounds | "white"

We've also updated Breadcrumb background setting to accept `"default"` as a background color (and for "default" to be the default!). We may add this functionality to other components in the future. When the background color is to `"default"`, the component will get the same background color as the site `body` (as `$theme-body-background-color`, specifically). While this isn't the same as using `transparent`, it can have a somewhat similar effect, and prevent you from having to set a specific background color twice, just set Breadcrumb to "default" and whatever you set for `$theme-body-background-color` will apply to Breadcrumb as well.

It's a tradeoff. We just can't make a realistic assumption that a transparent background always equals the body background, and it's too valuable to be able to check for accessible contrast and update colors automatically. While transparent backgrounds could be useful, for now we have to continue to throw errors when we see that value passed.
